junebug/config.py

```diff
@@ -34,3 +34,7 @@ class JunebugConfig(Config):
             'username': 'guest',
             'password': 'guest'
         })
+
+    ttl = ConfigInt(
+        "Maximum time allowed to reply to messages",
+        default=60)
```

Comment:
Hmm. I'd imagine it would have the same ttl value, but maybe for some reason we want to set one of the ttls explicitly?